daemon/state.go

```diff
@@ -134,6 +133,11 @@ func (d *Daemon) regenerateRestoredEndpoints(state *endpointRestoreState) {
 	for _, ep := range state.restored {
 		go func(ep *endpoint.Endpoint, epRegenerated chan<- bool) {
 			ep.Mutex.Lock()
+
+			// Insert into endpoint manager so it can be regenerated when calls to
+			// TriggerPolicyUpdates() are made.
+			endpointmanager.Insert(ep)
```

Comment:
The only issue I see with this is the `cilium endpoint list` can be empty as soon the `Endpoints restored, endpoints will be regenerated in background` is printed and the endpoints will show up afterwards. Might confuse users, no?

Reply:
I'll remove the 'regenerated in background' part of the message.

Reply:
Discussed and resolved offline.